xml/bp_troubleshooting_report.xml

```diff
@@ -27,14 +27,41 @@
   </para>
 <screen>&prompt.user;rpm -q supportutils-plugin-ses</screen>
   <para>
-   If it is missing on the &ceph; server, install it with
+   If it is missing on the &ceph; server, install it with:
   </para>
 <screen>&prompt.root;zypper ref &amp;&amp; zypper in supportutils-plugin-ses</screen>
+  <para>
+    <literal>supportutils-plugin-ses</literal> will try to capture as much
+    useful information as it can about both the &ceph; cluster as a whole,
+    and about the host on which it is running. Some information, including
+    <command>ceph daemon status</command>, podman image information, rpm verification and some
+    relevant configuration data will be saved to the <filename>plugin-ses.txt</filename> file
+    inside the <command>supportconfig</command> tar archive. A lot more information, including
```

Comment:
Is `supportconfig` a command or a filename? I would have guessed, if we're talking about a filename, that a <filename>...</filename> tag would be more appropriate here?

Reply:
I'm just using what was consistently used above, I haven't made this decision :woman_shrugging: